### PR TITLE
Add Vietnamese Elephant with Monochord

### PR DESCRIPTION
**Pull request by hybridherbst**
This is a production asset from [Songs of Cultures](https://songsofcultures.com), designed for QuickLook, using skeletal animation and UsdAudio.  
I was kindly allowed to bring it here under CC-BY-SA by [A.MUSE – Interactive Design Studio](https://amuse.vision/).

Readme link for quick previewing: https://github.com/prefrontalcortex/usd-wg-assets/tree/soc-elephant/full_assets/ElephantWithMonochord
A live version can be seen here: https://songsofcultures.com/en/ar-collection.

Note: the Readme contains a direct AR-capable link that necessarily points to the fork right now; that would need to be updated while or after merging to the direct link from `usd-wg/assets`.

### CODE REVIEW

File: full_assets/ElephantWithMonochord/Readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+_Video with Audio_
+
+## Screenshot
+
+![omniverse](screenshots/20220604-omniverse.png)  
+_Oniverse CREATE with skeleton hierarchies and UsdAudio_
+
+## Augmented Reality
+
+Open this page on iOS and click on the image:
```
Comment (meshula) [inline full_assets/ElephantWithMonochord/Readme.md:15]: I downloaded the asset, and hit spacebar on a mac ~ it played properly with sound and without transform or rendering errors. Do we want to say anything about mac quicklook?

File: full_assets/ElephantWithMonochord/Readme.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+It contains both Skeletal Animation on two meshes (elephant and monochord) as well as [UsdAudio](https://graphics.pixar.com/usd/release/wp_usdaudio.html) data.  
+
+## Shading
+
+The model uses the same texture for diffuseColor and emission. This creates an apperance of being "unlit", with no actual Unlit shading model currently being available in QuickLook.  
+
+## Problem: Only one mesh plays skeletal animation
+
+ ![usdview](screenshots/20220604-usdview-missing-skinned-animation.gif)  
+ _The monochord doesn't play the correct skinned animation here._
```
Comment (meshula) [inline full_assets/ElephantWithMonochord/Readme.md:31]:
```suggestion
 _The monochord doesn't play the correct skinned animation in usdview._
```

File: full_assets/ElephantWithMonochord/Readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Known Issue: Bounds calculations
+
+This model may have incorrect extents. Some viewers may calculate new mesh bounds from the actual skinned data / bones, others may use the provided extents data and thus show only a small part by default.  
+
+![usdview](screenshots/20220604-usview-wrong-bounds.png)
```
Comment (meshula) [inline full_assets/ElephantWithMonochord/Readme.md:37]: asset filename has a typo

File: full_assets/ElephantWithMonochord/Readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+ ![usdview](screenshots/20220604-usdview-missing-skinned-animation.gif)  
+ _The monochord doesn't play the correct skinned animation here._
+
+## Known Issue: Bounds calculations
+
+This model may have incorrect extents. Some viewers may calculate new mesh bounds from the actual skinned data / bones, others may use the provided extents data and thus show only a small part by default.  
+
+![usdview](screenshots/20220604-usview-wrong-bounds.png)  
+_Extents leading to wrong start view. Note the small displayed box gizmo in the center of the model._
```
Comment (meshula) [inline full_assets/ElephantWithMonochord/Readme.md:38]:
```suggestion
_Extents leading to wrong start view in usdview. Note the small displayed box gizmo in the center of the model._
```

**Review by meshula — CHANGES_REQUESTED** [review]
Very cute!

File: full_assets/ElephantWithMonochord/Readme.md (hunk 1 above)
Comment (hybridherbst) [reply full_assets/ElephantWithMonochord/Readme.md:15]: Added a note that this should also work on MacOS QuickLook
(side note: I don't think the Mac and iOS versions are identical)

**Review by meshula — APPROVED** [review]
Thanks for the word smithing. Looks good to me!

**Pull request merged**